### ISSUE TITLE
Add option to use oembed

### ISSUE THREAD

**Issue by camilleroux** [gottfrois/link_thumbnailer#112]
[oembed](http://oembed.com/) is a format for allowing an embedded representation of a URL on third party sites. It could be a good add to the library!

**Comment by gottfrois** [comment]
Absolutely! Should be quite easy to add since it comes down to implementing a new set of "scrapers".

BTW you can implement those scrapers on your own project and inject them using the proper gem config: https://github.com/gottfrois/link_thumbnailer/blob/master/lib/link_thumbnailer/scraper.rb#L36
